src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7621,6 +7621,13 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public override string ToString() { throw null; }
         public string ToString(int startIndex, int length) { throw null; }
+        public ChunkEnumerator EnumerateChunks() { throw null; }
```

Comment:
From the API review I thought this was supposed to be called GetChunks?

Reply:
You're right.